src/Sulu/Bundle/AdminBundle/Resources/js/components/Backdrop/Backdrop.js

```diff
@@ -26,6 +33,14 @@ export default class Backdrop extends React.PureComponent {
             [backdropStyles.backdrop]: true,
             [backdropStyles.isVisible]: isVisible,
         });
+        const body = document.body;
```

Comment:
Yes. I did a quick search. However, even the styleguidist project itself does nothing special about modals: https://github.com/styleguidist/react-styleguidist/tree/master/examples/basic/src/components/Modal